api/client/client.go

```diff
@@ -31,9 +31,12 @@ type ListenEvent struct {
 }
 
 // NewClient starts the client
-func NewClient(host string, port int) (*Client, error) {
-	url := fmt.Sprintf("%v:%v", host, port)
-	conn, err := grpc.Dial(url, grpc.WithInsecure())
+func NewClient(maddr ma.Multiaddr) (*Client, error) {
```

Comment:
In keeping with using multiaddress everywhere...

Reply:
Yeh really nice, I like this. Should just get this into our heads as the default always.